lib/wings.rb

```diff
@@ -8,7 +8,19 @@ module Wings
 
 require 'valkyrie'
 require 'wings/resource_factory'
+require 'wings/metadata_adapter'
 require 'wings/valkyrizable'
 require 'wings/valkyrie_monkey_patch'
 
 ActiveFedora::Base.include Wings::Valkyrizable
+
+Valkyrie::MetadataAdapter.register(
+  Valkyrie::Persistence::ActiveFedora::MetadataAdapter.new,
```

Comment:
I think in this context this should be `Wings::Valkyrie::MetadataAdapter.new, :wings_adapter` here, since we're wanting to register the wings specific active fedora metadata adapter. Is that right?